addons/hr_expense/models/hr_expense.py

```diff
@@ -909,7 +910,7 @@ def _default_bank_journal_id(self):
         ('cancel', 'Refused')
     ], string='Status', index=True, readonly=True, tracking=True, copy=False, default='draft', required=True)
     payment_state = fields.Selection(
-        selection=lambda self: self.env["account.move"]._fields["payment_state"].selection,
+        selection=PAYMENT_STATE_SELECTION,
```

Comment:
Wouldn't this work too?
```suggestion
        selection=lambda self: self.env["account.move"]._fields["payment_state"]._description_selection(self.env),
```

This way it isn't tied to a static list and it works too with `selection_add` (and keeps the same translations without having to translate in `hr_expense`